test/unit/test_dss_docker_image.py

```diff
@@ -75,39 +75,49 @@ def test_entrypoint_default(facts):
 
 
 def test_entrypoint_with_copy_args():
-    jupyter = "/home/jupyter/jupyterenv/bin/jupyter-lab"
-    port = "port"
-    entrypoint = "/path/to/entrypoint.py"
-    initial = "/path/to/initial"
-    final = "/path/to/final"
-    user = "jupyter-user-name"
-    password = "jupyter-default-password"
-    logfile = "/path/to/jupyter-server.log"
     facts = {
         "dss_facts": {
+            "docker_group": "docker-group-name",
             "jupyter": {
-                "command": jupyter,
-                "port": port,
-                "user": user,
-                "password": password,
-                "logfile": logfile,
+                "command": "/home/jupyter/jupyterenv/bin/jupyter-lab",
+                "port": "port",
+                "user": "jupyter-user-name",
+                "group": "jupyter-group-name",
+                "home": "/home/user",
+                "password": "jupyter-default-password",
+                "logfile": "/path/to/jupyter-server.log",
             },
-            "entrypoint": entrypoint,
+            "entrypoint": "/path/to/entrypoint.py",
             "notebook_folder": {
-                "initial": initial,
-                "final": final,
+                "initial": "/path/to/initial",
+                "final": "/path/to/final",
             }}}
-    assert create_image.entrypoint(facts) == [
-        "python3",
-        entrypoint,
-        "--notebook-defaults", initial,
-        "--notebooks", final,
-        "--jupyter-server", jupyter,
-        "--port", port,
-        "--user", user,
-        "--password", password,
-        "--jupyter-logfile", logfile,
-    ]
+
+    def fact(*args):
+        return create_image.get_fact(facts, *args)
+
+    expected = {
```

Comment:
This can't be a dict, because the order of python3 and entrypoint is important only the parameter afterwards can be different order.